packages/relay/src/lib/eth.ts

```diff
@@ -503,6 +503,13 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} getBalance(account=${account}, blockNumberOrTag=${blockNumberOrTag})`);
 
+    if (!EthImpl.blockTagIsLatestOrPending(blockNumberOrTag)) {
```

Comment:
Q: What's the need for this?
Add a comment.

Reply:
Nice, good find.

Reply:
Also created this ticket which will address this issue https://github.com/hashgraph/hedera-mirror-node/issues/4762
Hopefully we can focus on this in the next sprint

Reply:
Nice